runtime/src/lib.rs

```diff
@@ -1352,6 +1353,69 @@ impl thea_executor::Config for Runtime {
+	type MaxAuthorities = MaxAuthorities;
+	type Executor = TheaExecutor;
+	type WeightInfo = thea_message_handler::weights::WeightInfo<Runtime>;
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+construct_runtime!(
+	pub enum Runtime where
+		Block = Block,
+		NodeBlock = polkadex_primitives::Block,
+		UncheckedExtrinsic = UncheckedExtrinsic
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
+		Utility: pallet_utility::{Pallet, Call, Event} = 1,
+		Babe: pallet_babe::{Pallet, Call, Storage, Config, ValidateUnsigned} = 2,
+		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
+		Authorship: pallet_authorship::{Pallet, Storage} = 4,
+		Indices: pallet_indices::{Pallet, Call, Storage, Config<T>, Event<T>} = 5,
+		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 6,
+		TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>} = 7,
+		ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Pallet, Call, Storage, Event<T>, ValidateUnsigned} = 8,
+		Staking: pallet_staking::{Pallet, Call, Config<T>, Storage, Event<T>} = 9,
+		Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 10,
+		Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 11,
+		TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 12,
+		Elections: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>, Config<T>} = 13,
+		TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 14,
+		Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event, ValidateUnsigned} = 15,
+		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 16,
+		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 17,
+		ImOnline: pallet_im_online::{Pallet, Call, Storage, Event<T>, ValidateUnsigned, Config<T>} = 18,
+		AuthorityDiscovery: pallet_authority_discovery::{Pallet, Config} = 19,
+		Offences: pallet_offences::{Pallet, Storage, Event} = 20,
+		Historical: pallet_session_historical::{Pallet} = 21,
+		Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 22,
+		Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>} = 23,
+		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 24,
+		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 25,
+		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 26,
+		Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 27,
+		OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 28,
+		PDEXMigration: pdex_migration::pallet::{Pallet, Storage, Call, Event<T>, Config<T>} = 29,
+		Democracy: pallet_democracy::{Pallet, Call, Storage, Event<T>, Config<T>} = 30,
+		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 31,
+		//RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 32,
```

Comment:
Should be removed?
```suggestion
```

Reply:
No, until we finalize this change is okay, let it be there,